PendingReleaseNotes

```diff
@@ -73,10 +73,6 @@
   or use any other convenient way to restore the schedule after the
   upgrade.
 
-
->=16.0.0
```

Comment:
@tchaikov why are we removing this?

Reply:
we have two `>=16.0.0` in this very file. the first occurrence is at the first line. i assume that there is no point repeating it.

Reply:
absolutely!